app/assets/javascripts/controllers/IPSScheduleCtrl.js

```diff
@@ -22,7 +21,6 @@ planner.controller('IPSScheduleCtrl', ['$scope', '$rootScope', 'planService', '$
       $timeout(function() {
         $scope.years.push($scope.possibleYears[nextYear]);
         nextYear++;
-        Flash.create('success', messages.newYear);
       }, 150);
     }
```

Comment:
We've been cautioned against arbitrary ms in timeout calls, and timeout calls in general. Is this for throttling purposes?
Try removing the timeout. Shouldn't hurt.